webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Bios/index.js

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../../../../../common/I18n';
+import CardTemplate from '../../../../Templates/CardItem/CardTemplate';
+import SkeletonLoader from '../../../../../common/SkeletonLoader';
+import DefaultLoaderEmptyState from '../../../../DetailsCard/DefaultLoaderEmptyState';
+import { STATUS } from '../../../../../../constants';
+
+const BiosCard = ({ status, hostDetails }) => {
+  const { facts } = hostDetails;
+
+  const getValue = key => facts[key];
+
+  return (
+    <CardTemplate header={__('BIOS')} expandable masonryLayout>
+      <DescriptionList isCompact isHorizontal>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('Vendor')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <SkeletonLoader
+              status={status}
+              emptyState={<DefaultLoaderEmptyState />}
+            >
+              {facts !== undefined
+                ? getValue(
+                    Object.keys(facts).find(
+                      k => k.includes('vendor') && k.includes('bios')
+                    )
+                  )
+                : undefined}
```

Comment:
As noted in https://projects.theforeman.org/issues/35696, we should really normalize the facts into reported data and use that instead of directly reading the raw facts.